client/py_client/ipfs.py

```diff
@@ -1,16 +1,23 @@
 import subprocess
 import re
 import warnings
+import os
 
-ICONS_PATH = '../test-data/icons'
+local_ipfs_key = None
 
+if os.path.isfile("client\py_client\config.py"):
```

Comment:
I don't get that. Why do you load that?

Reply:
it was to use the ipfs_api_key locally from a config file. now i export the env variable and removed the logic for reading the key from a local file. -> done